packages/docs/src/pages/changelog.tsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import Layout from "@theme/Layout";
+
+import styles from "./changelog.module.css";
+import ChangelogScreen, { ChangelogItem } from "../components/Changelog";
+import LearnMoreFooter from "../components/LearnMore/LearnMoreFooter";
+import HomepageButton from "../components/HomepageButton";
+import LinkButton from "../components/LinkButton";
+
+// content field supports markdown
+const changelog: ChangelogItem[] = [
+  {
+    version: "1.5.x",
+    date: "2025-04-11",
+    title: "Radon AI",
+    content: `
+- [**Radon AI Chat**](/docs/features/radon-ai): a GitHub Copilot Chat participant enhanced with up-to-date information about the React Native ecosystem. At the heart of it is our extensive React Native knowledge database, which is queried before answering your question.
+
+<video autoPlay loop width="700" controls className="shadow-image changelog-item">
+  <source src="/video/ide_chat.mp4" type="video/mp4"/>
+</video>
+
+
+- **Improved EAS workflows**: You'll now see clearer build error messages directly, and all your EAS settings in launch configs are consistently recognized. Plus, automatic build fetching will show you only relevant builds.
+
+- **Home / App Switch buttons support**: Added options in the device settings menu that can trigger Home and AppSwitch button presses on active device.
+
+- Improved error & exception handling with clickable links
+
+- Support for React Native 0.79
+`,
+    JSX: (
+      <LinkButton
+        title="See v1.5.0 release notes on GitHub"
+        href="https://github.com/software-mansion/radon-ide/releases/tag/v1.5.0"
+      />
+    ),
+  },
+  {
+    version: "1.4.x",
+    date: "2025-03-10",
+    title: "Zero-config React Query devtools, Outline Renders",
+    content: `
+- [**Outline Renders**](/docs/features/dev-tools#outline-renders-react-scan): [react-scan](https://react-scan.com/) integration that help you visualize React renders happening within the app.
```

Comment:
I personally don't like saying we have "react-scan integration", since we support only this single feature from the tool which can do a lot more for you.
I'm not saying we should change this, I just feel like it might give unreasonable expectations that `react-scan` actually works with RN apps in Radon somehow.
Maybe somehow underline that we have the "render outlines" from "react-scan", not we have "react-scan" which enables "render outlines"

Reply:
Fixed in [e04c4d5](https://github.com/software-mansion/radon-ide/pull/1105/commits/e04c4d5cc1f82b08440c87a52f561e76ee416053) 👍